src/illumidesk/setup_course/course.py

```diff
@@ -160,19 +160,14 @@ def run(self):
         grader service.
         """
         logger.debug('Running grader container with exchange root %s' % self.exchange_root)
-        jupyterhub_api_url = os.environ.get('JUPYTERHUB_API_URL')
-        jupyterhub_api_token = os.environ.get('JUPYTERHUB_API_TOKEN')
-        logger.debug('Grader container JUPYTERHUB_API_URL set to %s' % jupyterhub_api_url)
-        logger.debug('Grader container JUPYTERHUB_API_TOKEN set to %s' % jupyterhub_api_token)
         self.client.containers.run(
             detach=True,
             image=os.environ.get('GRADER_SERVICE_IMAGE'),
             command=['start-notebook.sh', f'--group=formgrade-{self.course_id}'],
             environment=[
                 f'JUPYTERHUB_SERVICE_NAME={self.course_id}',
                 f'JUPYTERHUB_API_TOKEN={self.token}',
-                f'JUPYTERHUB_API_URL={jupyterhub_api_url}',
-                f'JUPYTERHUB_BASE_URL=https://{self.domain}/',
+                f'JUPYTERHUB_API_URL={self.jupyterhub_api_url}',
```

Comment:
If you remove this variable definition then the code won't run in the new line 170, right?

Reply:
Good catch.